projects/Respiration_Fluxnet/Q10estimation_new.jl

```diff
@@ -0,0 +1,231 @@
+# =============================================================================
+# Setup and Data Loading
+# =============================================================================
+using Pkg
+Pkg.activate("projects/Respiration_Fluxnet")
+Pkg.develop(path=pwd())
+
+manifest_path = joinpath(pwd(), "Manifest.toml")
+if !isfile(manifest_path)
+    Pkg.instantiate()
+end
+
+# start using the package
+using EasyHybrid
+using AxisKeys
+
+include("Data/load_data.jl")
+
+# =============================================================================
+# Load data
+# =============================================================================
+
+# copy data to data/data20240123/ from here /Net/Groups/BGI/work_4/scratch/jnelson/4Sinikka/data20240123
+# or adjust the path to /Net/Groups/BGI/work_4/scratch/jnelson/4Sinikka/data20240123 + FluxNetSite
+
+site = load_fluxnet_nc(joinpath(@__DIR__, "Data", "data20240123", "US-SRG.nc"), timevar="date")
+
+site.timeseries.dayofyear = dayofyear.(site.timeseries.time)
+site.timeseries.sine_dayofyear = sin.(site.timeseries.dayofyear)
+site.timeseries.cos_dayofyear = cos.(site.timeseries.dayofyear)
+
+# explore data structure
+println(names(site.timeseries))
+println(site.scalars)
+println(names(site.profiles))
+
+df = copy(site.timeseries[!, Not(:time, :date)])
+
+# =============================================================================
+# Targets, Forcing and Predictors definition
+# =============================================================================
+
+# Select target and forcing variables and predictors
+target_FluxPartModel = [:NEE]
+forcing_FluxPartModel = [:SW_IN, :TA]
+
+# Define predictors as NamedTuple - this automatically determines neural parameter names
+predictors = (Rb = [:SWC_shallow, :P, :WS, :sine_dayofyear, :cos_dayofyear], 
+              RUE = [:TA, :P, :WS, :SWC_shallow, :VPD, :SW_IN_POT, :dSW_IN_POT, :dSW_IN_POT_DAY])
+
+# =============================================================================
+# More Data Processing and Creation of KeyedArray
+# =============================================================================
+
+# Flatten predictors to get all unique column names
+all_predictor_cols = unique(vcat(values(predictors)...))
+col_to_select = unique([all_predictor_cols..., forcing_FluxPartModel..., target_FluxPartModel...])
+
+# select columns and drop rows with any NaN values
+sdf = copy(df[!, col_to_select])
+dropmissing!(sdf)
+
+for col in names(sdf)
+    T = eltype(sdf[!, col])
+    if T <: Union{Missing, Real} || T <: Real
+        sdf[!, col] = Float64.(coalesce.(sdf[!, col], NaN))
+    end
+end
+
+ds_keyed_FluxPartModel = to_keyedArray(Float32.(sdf))
+
+# =============================================================================
+# Parameter container for the mechanistic model
```

Comment:
do we want this to be defined in the project's directory, or should we move these definitions to `src/models/themodel`?